src/components/Exchange/PositionSeller.js

```diff
@@ -557,16 +664,23 @@ export default function PositionSeller(props) {
     let priceLimit = refPrice.mul(priceBasisPoints).div(BASIS_POINTS_DIVISOR);
     const minProfitExpiration = position.lastIncreasedTime + MIN_PROFIT_TIME;
     const minProfitTimeExpired = parseInt(Date.now() / 1000) > minProfitExpiration;
+
     if (nextHasProfit && !minProfitTimeExpired && !isProfitWarningAccepted) {
       if ((position.isLong && priceLimit.lt(profitPrice)) || (!position.isLong && priceLimit.gt(profitPrice))) {
         priceLimit = profitPrice;
       }
     }
+    const path = [tokenAddress0];
 
-    const withdrawETH = collateralTokenAddress === AddressZero || collateralTokenAddress === nativeTokenAddress;
+    if ((receiveToken?.address !== tokenAddress0) && allowReceiveTokenChange) {
+      path.push(receiveToken.address)
+    }
+
+    const withdrawETH = path.length === 1 
+      && (receiveToken.address === nativeTokenAddress || receiveToken.address === AddressZero)
```

Comment:
here you don't check if receiveToken is defined, however on line 675 you validate if `receiveToken` is defined. 

if it can be undefined we should check it here? or no need to check it on line 675?